prototypes/process-explorer/oss-frontend/src/app/generated-protos-files/ProcessExplorerMessages_pb.d.ts

```diff
@@ -0,0 +1,530 @@
+// package: 
```

Comment:
Are these files generated at build time? Should we push these files into the repo if they are generated at build time?

Reply:
We generate them from the proto file and yes we will have to push them since they provide a strongly typed interface to interact with data defined in the proto file